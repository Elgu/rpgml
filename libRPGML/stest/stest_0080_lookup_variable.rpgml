
int i = 23;
print( i );
print( "\n" );

Function f()
{
  return i;
}

{
  int i = 42;
  print( i );
  print( "\n" );
  print( .i );
  print( "\n" );

  print( f() );
  print( "\n" );
}

print( test_dir.s );
print( "\n" );

{
  Frame test_dir = Frame{ string s = "foo"; };

  print( test_dir.s );
  print( "\n" );

  print( .test_dir.s );
  print( "\n" );
}



