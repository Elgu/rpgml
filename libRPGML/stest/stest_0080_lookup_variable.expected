23
42
23
23
s
foo
s
